### PR TITLE
My subject for "Add another line to readme.md"

### PR DESCRIPTION
**Pull request by A025**
Something helpful goes here.

### CODE REVIEW

**Comment by A025** [comment]
> Something helpful goes here.

This is my answer for my questions

**Comment by A05** [comment]
It is necessary to add another new line to this nice document.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Sir.Cetus
 Repository for Cetus projects related to fishing activities.
 Another proposed line
+adding another line of text
```
Comment (A05) [inline README.md]: You should change this line of code.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # Sir.Cetus
 Repository for Cetus projects related to fishing activities.
```
Comment (A05) [inline README.md]: It looks good.

File: README.md (hunk 1 above)
Comment (A025) [reply README.md]: It will do it later!

**Pull request merged**